charts/elasticsearch/templates/nginx/nginx-es-configmap.yaml

```diff
@@ -38,29 +38,29 @@ data:
 
         # The following "location" rules limit airflow interactions to only their indices. Any further
         # additions should follow this pattern.
-        location ~* /_count$ {
+        location ~* ^/_count$ {
```

Comment:
I don't think this will work, as we need to redirect the `/_all/_count` form as well, that is used in provider versions [4.3.2+](https://airflow.apache.org/docs/apache-airflow-providers-elasticsearch/stable/index.html#id3).

Reply:
What about doing `^/(_all/)?_count` ?

Reply:
If you also cover `/*/_count` (literal star), should be fine. `_all` and `*` both mean all indices.

The good news is if it doesn't match, it just hits the deny block anyways.

Reply:
I tried to tackle this today but ran into the error `invalid number of arguments in "rewrite" directive in /etc/nginx/nginx.conf:25` when trying out:

```
        location ~* ^/(_all/|\*/)?_count$ {
          {{- if .Values.global.loggingSidecar.enabled }}
          rewrite ^/(_all/|\*/)?_count(.*) /vector.$remote_user.*/_count$1 break;
          {{- else }}
          rewrite ^/(_all/|\*/)?_count(.*) /fluentd.$remote_user.*/_count$1 break;
          {{- end }}
          proxy_pass http://elasticsearch;
        }
```

Reply:
I'm going to take this change out and maybe merge it in a next iteration.

Reply:
Sorry, I didn't mean to imply this is limited to _count, it applies so the other endpoints as well.

Reply:
You'll want to take out all of the anchoring changes.